### ISSUE TITLE
Allow spaces in `name` validation for user

### ISSUE THREAD

**Issue by sha1sum** [jeremykenedy/laravel-users#82]
It seems that a side effect of #65 was the inability to use spaces within names. Given that a user's full name is often what is used in this field (at least it is in all of our applications), this should probably be changed to use a regex as was noted in the conversation on #63. I am entering this issue for posterity, but will be forking and submitting a PR soon.

**Comment by jeremykenedy** [comment]
Usernames cannot have spaces

**Comment by sha1sum** [comment]
> Usernames cannot have spaces

Is this a design decision, or a statement of fact? Because certainly usernames can have spaces. If you're saying you don't want them to have spaces, then you're pretty much fine doing that; it's your package. But this is actually limiting what's already able to be done.

**Comment by jeremykenedy** [comment]
This is both.

**Comment by sha1sum** [comment]
I understand that you're worried about security. Closing my linked issue initially with "usernames cannot have spaces" has nothing to do with security. As far as I'm aware, spaces are not dangerous. There was no discussion at all, just closed with a single dismissive phrase. No effort to understand.

Spaces were all that was needed, but my PR allowed special characters, which you brought up only after my asking for more information. Again, not attempting to figure out why someone could possibly want the change, you again just dismissed it.

I'm aware that this is open source, and I thank you for that. I try to contribute where I can, and short, dismissive phrases like the ones I got from this attempt to contribute frankly discourage people from contributing in the future. We can get by without your assistance. I'm sorry that I tried to help improve things here.
